tests/components/plex/test_init.py

```diff
@@ -314,3 +314,25 @@ async def test_scan_clients_schedule(hass: HomeAssistant, setup_plex_server) ->
         await hass.async_block_till_done()
 
     assert mock_scan_clients.called
+
+
+async def test_setup_with_limited_credentials(
+    hass: HomeAssistant, entry, setup_plex_server
+) -> None:
```

Comment:
I moved this test out of `test_config_flow.py` as there is no reason to have it there...